vita3k/interface.cpp

```diff
@@ -440,6 +440,7 @@ static ExitCode load_app_impl(SceUID &main_module_id, EmuEnvState &emuenv, const
 
     LOG_INFO("{}: {}", emuenv.cfg[e_cpu_backend], emuenv.cfg.current_config.cpu_backend);
     LOG_INFO_IF(emuenv.kernel.cpu_backend == CPUBackend::Dynarmic, "CPU Optimisation state: {}", emuenv.cfg.current_config.cpu_opt);
+    LOG_INFO("Audio volume: {}", emuenv.cfg.current_config.audio_volume);
```

Comment:
exept for this, i no sur is really needed hmm 
i have  set this log for set important log for debug, but so for volume hmm

Reply:
I put it in here so that if someone comes along with a problem like “I have no sound”, one look at the log would tell you that they might just have their volume set to 0. I can remove the log line though if it’s not wanted.

Reply:
hmm I see, indeed;, well not sure that it happens lol
But I'll let Macdu decide :)

Reply:
I agree with Zangetsu here, I don't think it's necessary to put it in the log.

Reply:
Okay, removed.